node-repository/src/main/java/com/yahoo/vespa/hosted/provision/node/Agent.java

```diff
@@ -21,6 +21,7 @@
     InactiveExpirer,
     ProvisionedExpirer,
     ReservationExpirer,
+    ParkedExpirer,
```

Comment:
Must be added to `com.yahoo.vespa.hosted.controller.api.integration.noderepository.NodeHistory` as well.

Reply:
You can disregard this as #18201 should take care of it.

Reply:
It was already there